docs/components/tuner/callbacks.md

```diff
@@ -0,0 +1,67 @@
+# Callbacks
+
+Callbacks offer a way to integrate various auxiliary tasks into the training loop. We already offer built-in callbacks for some common tasks, such as
```

Comment:
I would remove `already` from the latter sentence. That might make the sentence age better. If Jina exists for 5 years, the features will have outgrown the mention `already` here.